sklearn/cluster/_bicluster.py

```diff
@@ -319,55 +319,55 @@ class SpectralBiclustering(BaseSpectral):
 
     Parameters
     ----------
-    n_clusters : integer or tuple (n_row_clusters, n_column_clusters)
+    n_clusters : integer or tuple (n_row_clusters, n_column_clusters), default=3
         The number of row and column clusters in the checkerboard
         structure.
 
-    method : string, optional, default: 'bistochastic'
+    method : string, default='bistochastic'
         Method of normalizing and converting singular vectors into
         biclusters. May be one of 'scale', 'bistochastic', or 'log'.
         The authors recommend using 'log'. If the data is sparse,
         however, log normalization will not work, which is why the
         default is 'bistochastic'. CAUTION: if `method='log'`, the
```

Comment:
Since we are changing the doctstring could you make the following change:

```suggestion
        default is 'bistochastic'.
        
        .. warning::
           if `method='log'`, the data must be sparse.
```